libs/SalesforceAnalytics/src/com/salesforce/androidsdk/analytics/logger/FileLogger.java

```diff
@@ -102,11 +108,11 @@ public int getMaxSize() {
      *
      * @param size Maximum number of log lines.
      */
-    public synchronized void setMaxSize(int size) {
+    public void setMaxSize(int size) {
         if (size < 0) {
             size = 0;
         }
-        maxSize = size;
+        storeFileLoggerPrefs(size);
```

Comment:
Shouldn't maxSize also be set?
If I construct a FileLogger then call setMaxSize, woudn't the chosen maxSize only be used the next time I construct a FileLogger?